pep-0009.txt

```diff
@@ -5,225 +5,224 @@ Last-Modified: $Date$
 Author: Barry Warsaw <barry@python.org>
 Status: Withdrawn
 Type: Process
-Content-Type: text/plain
+Content-Type: text/x-rst
```

Comment:
`Title: Sample Plaintext PEP Template` -> `Title: Sample reStructuredText PEP Template` ?

Reply:
Sample Plaintext PEP  template is correct.

Sample restructuredText PEP template is [PEP 12](https://www.python.org/dev/peps/pep-0012/) :)